src/scripts.js

```diff
@@ -9,27 +9,24 @@ import './images/search.png';
 import './images/cookbook.png';
 import User from './user';
 import Recipe from './recipe';
+import domUpdates from './domUpdates';
 
 // ************ QUERY SELECTORS ***************
 let allRecipesBtn = document.querySelector(".show-all-btn");
 let filterBtn = document.querySelector(".filter-btn");
-let fullRecipeInfo = document.querySelector(".recipe-instructions");
 let main = document.querySelector("main");
 let pantryBtn = document.querySelector(".my-pantry-btn");
 let savedRecipesBtn = document.querySelector(".saved-recipes-btn");
 let searchBtn = document.querySelector(".search-btn");
 let searchForm = document.querySelector("#search");
-let searchInput = document.querySelector("#search-input");
 let showPantryRecipes = document.querySelector(".show-pantry-recipes-btn");
-let tagList = document.querySelector(".tag-list");
 let modifyPantryBtn = document.querySelector(".modify-pantry-btn")
-let searchIngBtn = document.querySelector(".search-ingredients-btn")
+let searchIngredientBtn = document.querySelector(".search-ingredients-btn")
```

Comment:
Thank you for fixing these variable names. Much better and more descriptive!

Reply:
Oh yes, I didn't even notice that before. Now it is much easier to see which elements we are targeting